app/scripts/controllers/metamask-notifications/constants/notification-schema.ts

```diff
@@ -33,6 +33,16 @@ export const NOTIFICATION_CHAINS = {
   LINEA: '59144',
 };
 
+export const CHAIN_SYMBOLS = {
+  [NOTIFICATION_CHAINS.ETHEREUM]: 'ETH',
+  [NOTIFICATION_CHAINS.OPTIMISM]: 'ETH',
+  [NOTIFICATION_CHAINS.BSC]: 'BNB',
+  [NOTIFICATION_CHAINS.POLYGON]: 'MATIC',
+  [NOTIFICATION_CHAINS.ARBITRUM]: 'ETH',
+  [NOTIFICATION_CHAINS.AVALANCHE]: 'AVAX',
+  [NOTIFICATION_CHAINS.LINEA]: 'ETH',
+};
```

Comment:
Hardcoded Chain Symbols since we need Platform agnostic values (not tied to extension).

If there are shared values then sweet, otherwise we'll keep as is for now.